config/application.rb

```diff
@@ -1,10 +1,10 @@
 require_relative 'boot'
 
-# require 'action_controller/railtie'
-# require 'action_mailer/railtie'
-# require 'sprockets/railtie'
-# require 'rails/test_unit/railtie'
-require 'rails/all'
+require 'action_controller/railtie'
+require 'action_mailer/railtie'
+require 'sprockets/railtie'
+require 'rails/test_unit/railtie'
+# require 'rails/all'
```

Comment:
This I don't fully understand yet, however rails/all definitely gives me the idea that the kitchen sink was being requested, and that's why ActiveRecord was being used (which was what was preventing startup)